extensions/binding/org.eclipse.smarthome.binding.fsinternetradio.test/src/test/groovy/org/eclipse/smarthome/binding/fsinternetradio/test/FSInternetRadioHandlerOSGiTest.groovy

```diff
@@ -155,50 +166,38 @@ public class FSInternetRadioHandlerOSGiTest extends OSGiTest{
             assertThat ("The ThingStatus was not updated correctly when a wrong PIN is used", radioThingWithWrongPin.getStatus(), is(equalTo(ThingStatus.OFFLINE)))
             assertThat ("The ThingStatusInfo was not updated correctly when a wrong PIN is used", radioThingWithWrongPin.getStatusInfo().getStatusDetail(), is(equalTo(ThingStatusDetail.COMMUNICATION_ERROR)))
         }
-
-        ChannelUID powerChannelUID = powerChannel.getUID()
-        Item testItem = initializeItem(powerChannelUID, "${DEFAULT_TEST_ITEM_NAME}",
-                acceptedItemTypes.get(FSInternetRadioBindingConstants.CHANNEL_POWER))
-
-        waitForAssert {
-            assertThat ("The item's state was not updated correctly when a wrong PIN is used", testItem.getState(), is (UnDefType.NULL))
-        }
     }
 
     @Test
-    void 'verify OFFLINE Thing-status and NULL Item-state when the HTTP response cannot be parsed correctly'() {
+    void 'verify OFFLINE Thing status when the HTTP response cannot be parsed correctly'() {
 
         // create a thing with two channels - the power channel and any of the others
         String modeChannelID = FSInternetRadioBindingConstants.CHANNEL_MODE
         String acceptedItemType = acceptedItemTypes.get(modeChannelID)
         createChannel(DEFAULT_THING_UID, modeChannelID, acceptedItemType)
 
         Thing radioThing = initializeRadioThing(DEFAULT_COMPLETE_CONFIGURATION)
-        testRadioThingWithConfiguration(radioThing)
+        testRadioThingConsideringConfiguration(radioThing)
 
         // turn-on the radio
         turnTheRadioOn(radioThing)
 
+        ChannelUID modeChannelUID = radioThing.getChannel(modeChannelID).getUID()
+        Item modeTestItem = initializeItem(modeChannelUID, "mode", acceptedItemType)
```

Comment:
You are right, I don't need the item. That line is deleted and the PR is updated.